Dockerfile

```diff
@@ -1,3 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
This was missed, which reminds me, please add skywalking-eyes check for BanyanDB repo

https://github.com/apache/skywalking-eyes

Reply:
Yes. We have `skywalking-eyes` check for `Check` build.

Reply:
> `pre-push: generate lint license-check check ## Check source files before pushing to the remote repo`

This requires everyone to run `make pre-push` before pushing codes, but it may be forgot, it's better to add the task in to CI workflow